packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -136,12 +147,27 @@ const ActivityDetailsSidebar = (props: Props) => {
   } = useDialogState()
 
   const [selectedUsers, setSelectedUsers] = React.useState<Option[]>([])
+  const selectedUser = selectedUsers[0]
+  const {organizations: viewerOrganizations} = viewer
+
+  const viewerOrganizationIds = new Set(viewerOrganizations.map((org) => org.id))
+
+  const selectedUserOrganizationIds = selectedUser?.organizationIds ?? []
+  const mutualOrgs = selectedUserOrganizationIds.filter((orgId) => viewerOrganizationIds.has(orgId))
```

Comment:
+1 `mutualOrgs` here is actually `mutualOrgIds`, but I'd actually suggest you refactor this so that this holds org objects, rather than just renaming:
```suggestion
  const selectedUserOrganizationIds = new Set(selectedUser?.organizationIds ?? [])
  const mutualOrgs = viewerOrganizations.filter((orgId) => selectedUserOrganizationIds.has(orgId))
```